docs/genesis.md

```diff
@@ -46,6 +46,25 @@ The below defines all hardcoded values set in the geneses:
 
 <!-- markdownlint-disable MD013 -->
 
+### Predeployed accounts node genesis
+
+<!-- markdownlint-disable MD013 -->
+
+| Contract address | Class hash                                                         | Description                     | Optional private key |
+| :--------------- | :----------------------------------------------------------------- | :------------------------------ | :------------------- |
+| 0x1              | 0x0279d77db761fba82e0054125a6fdb5f6baa6286fa3fb73450cc44d193c2d37f | No Validation Account           | null                 |
+| 0x2              | 0x06f0d6f6ae72e1a507ff4b65181291642889742dbf8f1a53e9ec1c595d01ba7d | Argent Account                  | `pk`                 |
+| 0x3              | 0x006280083f8c2a2db9f737320d5e3029b380e0e820fe24b8d312a6a34fdba0cd | Openzeppelin Account            | `pk`                 |
```

Comment:
Both have the same private key?

Reply:
I believe they do, if I understood the `print_development_accounts` correctly [there](https://github.com/keep-starknet-strange/madara/blob/e9d77e31f669e028f63edfc95cdc10858d96490e/crates/node/src/chain_spec.rs#L102).